projects/ion/src/lib/info-badge/info-badge.component.scss

```diff
@@ -0,0 +1,69 @@
+@import '../../styles/index.scss';
+
+@mixin set-size($width, $height, $padding) {
+  width: $width;
+  height: $height;
```

Comment:
width and heigth foreve is same, you can change to only one parameter. Size.

Reply:
Done :D